lib/situation_room/site_controller.ex

```diff
@@ -0,0 +1,89 @@
+defmodule SituationRoom.Site.Controller do
+  @moduledoc """
+  Documentation for `SituationRoom.Site.Controller`.
+  """
+  alias SituationRoom.Site
+  use Plug.Router
+
+  plug(:match)
+
+  plug(Plug.Parsers,
+    parsers: [:urlencoded, :multipart],
+    pass: ["text/*"]
+  )
+
+  plug(:dispatch)
+
+  # Get all sites
+  get "/" do
+    send_resp(
+      conn,
+      200,
+      "[#{for content <- Site.get_all_sites(), do: "#{build_site_resp(content)}, "}]"
+    )
+  end
+
+  # Get a site by id
+  get "/:id" do
+    if not is_integer(conn.params["id"]) do
```

Comment:
Pretty sure this won't ever be an integer as the query params and URLs are always strings would would need to be converted first.
That also shouldnt matter though.  We should just return a 404 if an invalid `:id` value is sent

    GET /sites/abc => 404 Not Found
    GET /sites/-12sabc => 404 Not Found
    GET /sites/123 => 200 with Site attached